application/world/services/auto_bible_generator.py

```
@@ -740,16 +823,29 @@ def _summarize_worldbuilding(self, wb: Dict[str, Any]) -> str:
+        # 强制增加约束，禁止输出思考过程，避免 DeepSeek-R1 等模型输出乱码
+        system_prompt += (
+            "\n\n【极其严格的 JSON 格式约束】"
+            "\n1. 绝对不要输出 <think> 标签或任何思考过程！"
+            "\n2. 绝对不要在 JSON 的字符串值内部使用双引号（\"）！如果需要强调或引用，请务必使用单引号（'），例如 '中关村'。"
+            "\n3. 请直接返回合法的 JSON，不要附加任何 Markdown 标记或多余的文字说明。"
+        )
+
         prompt = Prompt(system=system_prompt, user=user_prompt)
-        config = GenerationConfig(max_tokens=2048, temperature=0.7)
+        config = GenerationConfig(max_tokens=8192, temperature=0.7)
         print(f"[DEBUG] _call_llm_and_parse: Calling LLM service", file=sys.stderr, flush=True)
```

Ruff (0.15.10)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)